data/governments.txt

```diff
@@ -131,6 +131,62 @@ government "Free Worlds"
 	"hostile hail" "hostile free worlds"
 	raid "pirate raid"
 
+government Gegno
+	swizzle 28
+	color 0.56 0.44 0.2
+	"player reputation" 0.01
+	"crew attack" 1.7
+	"crew defense" 2.2
+	"attitude toward"
+		"Gegno Scin" 1
+		"Gegno Vi" 1
+		Quarg 0.1
+	"penalty for"
+		assist 0
+		board 1
+		capture 1
+		destroy 1
+		atrocity 1
+		provoke 1
```

Comment:
What's the reason for these custom penalty values? If you don't want the player to gain reputation by assisting them, then you only need

```
"penalty for"
	assist 0
```

Reply:
I want the player to lose reputation for the rest. The values being positive here is a loss.

Reply:
I mean by default boarding is less of a penalty than destroying which is less than atrocity. Why everything the same or why not leave the defaults?

Reply:
Ah, I didn't realize defaults were measured that way. So if I remove these, the game defaults them?

Reply:
Does this include provoke?

Reply:
Yeah, since you only need assist I presume which is why I suggested only specifying assist (the rest will default).